src/query/service/tests/it/storages/fuse/operations/navigate.rs

```diff
@@ -84,7 +84,12 @@ async fn test_fuse_navigate() -> Result<()> {
     assert_eq!(second_snapshot, loc);
     let version = TableMetaLocationGenerator::snapshot_version(loc.as_str());
     let snapshots: Vec<_> = reader
-        .snapshot_history(loc, version, fuse_table.meta_location_generator().clone())
+        .snapshot_history(
+            fuse_table.get_operator(),
```

Comment:
The same question. Can we init the reader with op?

Reply:
I'm working on this.